docs/topics/settings.rst

```diff
@@ -1116,6 +1116,29 @@ Default: ``'scrapy.spiderloader.SpiderLoader'``
 The class that will be used for loading spiders, which must implement the
 :ref:`topics-api-spiderloader`.
 
+.. setting:: SPIDER_LOADER_WARN_ONLY
+
+SPIDER_LOADER_WARN_ONLY
+-----------------------
+
+.. versionadded:: 1.3.3
```

Comment:
it should be probably changed in master after the merge (it is `versionadded:: 1.4` in master)